README.rst

```diff
@@ -956,6 +956,8 @@ Choose from the list of available rules:
   - ``include`` (``array``): list of additional constants to fix; defaults to ``[]``
   - ``scope`` (``'all'``, ``'namespaced'``): only fix constant invocations that are made
     within a namespace or fix all; defaults to ``'all'``
+  - ``strict`` (``bool``): whether leading ``\`` of constant call not meant to have it
```

Comment:
if would be prefer `whether leading ``\`` of constant usage not meant to have it should be removed` as constants are not really called, WDYT?